### PR TITLE
[test/plugin] immigrate test zookeeper scenario. 

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
https://github.com/apache/skywalking/issues/3583#issuecomment-567320256

### CODE REVIEW

File: test/plugin/scenarios/zookeeper-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+# limitations under the License.
+
+registryItems:
+  applications:
+    - {zookeeper-scenario: 2}
+  instances:
+    - {zookeeper-scenario: 1}
+  operationNames:
+    - zookeeper-scenario: [Zookeeper/WatchedEvent/NodeDeleted, /zookeeper-scenario/case/zookeeper-case,
+                           Zookeeper/WatchedEvent/None, /zookeeper-scenario/case/healthCheck]
```
Comment (dmsolr) [inline test/plugin/scenarios/zookeeper-scenario/config/expectedData.yaml]: Why I cannot find the `Zookeeper/WatchedEvent/None` Segment in `SegmentItems`? 
Am I missing things?

File: test/plugin/scenarios/gateway-scenario/configuration.yml [hunk 2]
```diff
@@ -18,6 +18,6 @@ type: jvm
 entryService: http://localhost:8080/provider/b/testcase
 healthCheck: http://localhost:8080/provider/b/healthCheck
 startScript: ./bin/startup.sh
-framework: spring-cloud-gateway
```
Comment (dmsolr) [inline test/plugin/scenarios/gateway-scenario/configuration.yml]: You push the expected file, right?

File: test/plugin/scenarios/redisson-scenario/support-version.list [hunk 3]
```diff
@@ -14,40 +14,4 @@
...
-3.6.1
-3.6.0
-3.5.7
-3.5.6
-3.5.5
-3.5.4
-3.5.3
-3.5.2
\ No newline at end of file
+3.11.5
```
Comment (dmsolr) [inline test/plugin/scenarios/redisson-scenario/support-version.list]: Is this a mistake? :)

File: test/plugin/scenarios/zookeeper-scenario/support-version.list [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+3.5.6
+3.4.14
+3.4.9
+3.4.6
+3.4.3
+3.4.0
```
Comment (dmsolr) [inline test/plugin/scenarios/zookeeper-scenario/support-version.list]: These are 6 versions, not 8 versions.  please recheck the github-action configuration.

File: test/plugin/scenarios/zookeeper-scenario/config/expectedData.yaml (hunk 1 above)
Comment (zhaoyuguang) [reply test/plugin/scenarios/zookeeper-scenario/config/expectedData.yaml]: At begin, when `healthCheck` 
image
The delete method trigger `Zookeeper/WatchedEvent/None`,
So I do not assertion it

File: test/plugin/scenarios/gateway-scenario/configuration.yml (hunk 2 above)
Comment (zhaoyuguang) [reply test/plugin/scenarios/gateway-scenario/configuration.yml]: en. I fixed.

File: test/plugin/scenarios/redisson-scenario/support-version.list (hunk 3 above)
Comment (zhaoyuguang) [reply test/plugin/scenarios/redisson-scenario/support-version.list]: en. I fixed.

File: test/plugin/scenarios/zookeeper-scenario/support-version.list (hunk 4 above)
Comment (zhaoyuguang) [reply test/plugin/scenarios/zookeeper-scenario/support-version.list]: I has fixed.

File: test/plugin/scenarios/zookeeper-scenario/src/main/java/org/apache/skywalking/apm/testcase/zookeeper/controller/ZookeeperController.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        return "Success";
+    }
+
+    @RequestMapping("/healthCheck")
+    @ResponseBody
+    public String healthCheck() {
+        try {
+            zooKeeper.delete("/path", -1);
+        } catch (Exception e) {
+        }
```
Comment (arugal) [inline test/plugin/scenarios/zookeeper-scenario/src/main/java/org/apache/skywalking/apm/testcase/zookeeper/controller/ZookeeperController.java]: Hi, exception should be throw, not ignored. right?

File: test/plugin/scenarios/zookeeper-scenario/src/main/resources/log4j2.xml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    <Console name="Console" target="SYSTEM_OUT">
+        <PatternLayout pattern="%d [%traceId] %-5p %c{1}:%L - %m%n"/>
+    </Console>
+</Appenders>
+<Loggers>
+    <Root level="OFF">
+        <AppenderRef ref="Console"/>
+    </Root>
+</Loggers>
+</Configuration>
```
Comment (dmsolr) [inline test/plugin/scenarios/zookeeper-scenario/src/main/resources/log4j2.xml]: Note that, we will discard the `systemout` logs. If it is expected print, I suggest using the new log-configuration template.

Comment (zhaoyuguang) [reply test/plugin/scenarios/zookeeper-scenario/src/main/resources/log4j2.xml]: Like this?
```xml
<Configuration status="WARN">
    <Appenders>
        <Console name="Console" target="SYSTEM_ERR">
            <PatternLayout charset="UTF-8" pattern="[%d{yyyy-MM-dd HH:mm:ss:SSS}] [%p] - %l - %m%n"/>
        </Console>
    </Appenders>
    <Loggers>
        <Root level="WARN">
            <AppenderRef ref="Console"/>
        </Root>
    </Loggers>
</Configuration>
```

File: test/plugin/scenarios/zookeeper-scenario/src/main/java/org/apache/skywalking/apm/testcase/zookeeper/controller/ZookeeperController.java (hunk 5 above)
Comment (zhaoyuguang) [reply test/plugin/scenarios/zookeeper-scenario/src/main/java/org/apache/skywalking/apm/testcase/zookeeper/controller/ZookeeperController.java]: Right, I had fixed.

File: test/plugin/scenarios/zookeeper-scenario/config/expectedData.yaml (hunk 1 above)
Comment (zhaoyuguang) [reply test/plugin/scenarios/zookeeper-scenario/config/expectedData.yaml]: I had remove this unuse SegmentItem

**Pull request merged**